xapian-applications/omega/templates/query

```diff
@@ -4,31 +4,31 @@ $set{thousand,$.}$set{decimal,.}
 $setmap{prefix,,$split{ S B F},title,S,author,A,topic,B}
 ${
 $def{PREV,
-$if{$ne{$topdoc,0},<INPUT TYPE=image NAME="&lt;" ALT="&lt;"
-SRC="/icons/omega/prev.png" BORDER=0 HEIGHT=30 WIDTH=30>,
-<img alt="" src="/icons/omega/prevoff.png" height=30 width=30>}
+$if{$ne{$topdoc,0},<INPUT TYPE="image" NAME="&lt;" ALT="&lt;"
+SRC="/icons/omega/prev.png" BORDER="0" HEIGHT="30" WIDTH="30">,
+<img alt="" src="/icons/omega/prevoff.png" height="30" width="30">}
 }
 
 $def{NEXT,
-$if{$ne{$last,$msize},<input type=image name="&gt;" alt="&gt;"
-src="/icons/omega/next.png" border=0 height=30 width=30>,
-<img alt="" src="/icons/omega/nextoff.png" height=30 width=30>}
+$if{$ne{$last,$msize},<input type="image" name="&gt;" alt="&gt;"
+src="/icons/omega/next.png" border="0" height="30" width="30">,
+<img alt="" src="/icons/omega/nextoff.png" height="30" width="30">}
 }
 
-$def{P,<input type=image name="$1" value="$1" src="/icons/omega/page-$2.png" border=0$opt{a} alt="$1">}
+$def{P,<input type="image" name="$1" value="$1" src="/icons/omega/page-$2.png" border="$opt{a}" alt="$1">}
```

Comment:
You've misunderstood what `$opt{a}` is doing here — it's additional attributes, set (confusingly) further down in the document, at line 116. Quoting like this will result in something like `border=""` or `border=" HEIGHT=60 WIDTH=30"` or similar, neither of which is correct. You've made a similar mistake at line 100.

It's difficult to see how we can write automated tests that would catch this (although the one at line 100 would be easier). So what I'd suggest is that you write a manual test script which explains what to visually check for, ensuring that the different code paths are taken. (We might be able to automate some or all of this in future, with some cleverness.) Right now the changes are too many and too invasive to guarantee that code review is going to pick up any potential breakages, so I think we really need the manual test script for this.